contents/node_operator.rst

```diff
@@ -412,6 +412,101 @@ If the external node is running on IP 12.34.56.78 port 8545, the command would b
 
 You can also use ``https``. But keep in mind that Swarm *does not validate the certificate*.
 
+Connect to the SWAP-enabled testnet
+===================================
+The Swarm project now runs a SWAP-enabled (incentivized) testnet. It uses the same binary as the normal network node, but has incentivization switched on. 
+
+.. important::
+
+  The testnet is highly experimental and may be updated continuously until we release a mainnet version. All tokens used on this testnet are fictitious. Do NOT use this testnet with real value tokens. Funds can be lost.
+
+ 
+The first public incentives-enabled Swarm network runs on the ropsten network. 
+It runs with network id **5**
+
+Prerequisites
+-----------------
+* Create an account on **ropsten**
+* Fund your newly created account with test coins
```

Comment:
First step:
Create an account on Ropsten and get Ropsten Ethers (to pay transaction fees from a [Ropsten Faucet](https://faucet.metamask.io/)).
Second step: start Swarm with `--swap-skip-deposit`  flag and note the address of your newly deployed chequebook.
Third step: Fill in the address of the chequebook [here](https://ropsten.etherscan.io/address/0x49BF80bDEE2684580966e476aEe0DC3D773fFaf5#writeContract).

Reply:
Your logic would be correct, but the faucet is set up in such a way that it only funds chequebooks, hence the need to do it like this.

Reply:
thanks